oebue.xpr

```diff
@@ -31,7 +31,7 @@
                     </entry>
                     <entry>
                         <String>frameworks.directory.url.with.editor.variables</String>
-                        <String>file:/home/yannik/Documents/Oxygen%20Frameworks</String>
+                        <String>file:/C:/Users/uniwu/Documents/ediarum%20Frameworks</String>
```

Comment:
Wir sollten die oebue wohl rausnehmen. Hierzu mal ne gitignore reinschmeißen?

Reply:
Was meinst du genau mit oebue rausnehmen?

Reply:
Also meinst du dann eine .gitignore datei in der dann steht, dass oebue.xpr ignoriert werden soll?